sklearn/linear_model/tests/test_ransac.py

```diff
@@ -531,3 +531,15 @@ def test_ransac_final_model_fit_sample_weight():
     )
 
     assert_allclose(ransac.estimator_.coef_, final_model.coef_, atol=1e-12)
+
+
+def test_perfect_horizontal_line():
```

Comment:
Please add a comment to explain that this is a not regression test for `https://github.com/scikit-learn/scikit-learn/issues/16147`.

Reply:
Thanks a lot for your review. I add a comment to explain this is a regression test for #19497.
I consider your "not" as a typo. If it is not, could you explain me why it is not a regression test ?

Reply:
I think @ogrisel meant "a non-regression test", hence a typo.

Reply:
thanks